tshirts.c

```diff
@@ -17,6 +17,10 @@ int main() {
     assert(size(37) == 'S');
     assert(size(40) == 'M');
     assert(size(43) == 'L');
+    assert(size(38) == 'S');

```

Comment:
You are expecting the production code to classify Size 38 as both S and M, similarly for Size 42 too!!
Is this the right expectation?